concordium-contracts-common/src/schema_json.rs

```diff
@@ -956,13 +956,234 @@ impl Fields {
     }
 }
 
-/// Displays a pretty-printed JSON-template of the schema.
+/// Displays a pretty-printed JSON-template of the [`SchemaType`].
 impl Display for Type {
     fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
         write!(f, "{}", serde_json::to_string_pretty(&self.to_json_template()).unwrap())
     }
 }
 
+/// Displays the json template of the type indented.
+fn display_json_template_indented(
+    mut out: String,
+    type_schema: &Type,
+    type_schema_name: &str,
+    indent: usize,
+) -> String {
+    out = format!("{}{:>3$}{}:\n", out, "", type_schema_name, indent);
```

Comment:
Could you document what the `:>3$` does exactly?

Reply:
Explanation:

The value `""` is `right-aligned in width columns`. Meaning there is just `tabs/whitespace` created here.
https://doc.rust-lang.org/std/fmt/#fillalignment

The amount of `tabs/whitespace` created is defined in the third argument (`3$`), meaning the `indent` defines the amount of `tabs/whitespace` created.

Does the explanation clarify it? I would keep it as it is if ok since people could look up the meaning of the symbols e.g. here  `https://doc.rust-lang.org/std/fmt/#fillalignment`.

Reply:
Added some additional explanation about the `:>3$` syntax and also added the above link in a comment.